src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent.mdx

```diff
@@ -0,0 +1,181 @@
+</Callout>
+
+By default, the New Relic Prometheus agent uses two annotations to discover targets: `newrelic.io/scrape: "true"` and `prometheus.io/scrape: "true"`. All the targets in the cluster that are annotated with `newrelic.io/scrape: "true"` are discovered and scraped. However, the default annotation from the Prometheus community, `prometheus.io/scrape: "true"`, is discovered but we filter them to only scrape the [Prometheus integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro) that we support.
+
+[Prometheus Integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro) help you get curated experiences for services that expose metrics in Prometheus or Open Metrics format, bundling everything you need to start monitoring out of the box.
+
+For more information, see the [set up New Relic's Prometheus agent](/docs/integrations/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent) documentation.
+
+## Install only the Prometheus agent [#prometheus-install]
+
+If you don't need the Kubernetes integration, you can install the Prometheus agent by its own.
+
+1. Install the [Prometheus Agent](https://github.com/newrelic/newrelic-prometheus-configurator#readme) by running:
+  ```shell
+  helm repo add newrelic-prometheus https://newrelic.github.io/newrelic-prometheus-configurator
+  helm upgrade --install newrelic newrelic-prometheus/newrelic-prometheus-agent -f YOUR_CUSTOM_VALUES.yaml
+  ```
+
+2. To ensure the integration has been configured correctly, go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > Query builder**, and run this NRQL query to see if data has been reported:
+
+  ```sql
+  FROM Metric SELECT count(*) WHERE collector.name = 'prometheus-agent' since 1 hour ago
+  ```
+
+  <figcaption>
+    Install the quickstart containing the [dashboard for the Prometheus agent](https://newrelic.com/instant-observability/prometheus-agent).
+  </figcaption>
+
+## Install the dashboard [#io-dashboard]
+
+Regardless of whether you installed the Kubernetes integration or only the Prometheus agent, we also give you a curated dashboard for self-metrics with performance and health information, and also metrics volume sent.
+
+1. Install the [dashboard for the Prometheus agent](https://newrelic.com/instant-observability/prometheus-agent) in your New Relic account.
+
+<img
+  title="Quickstart dashboard for the Prometheus agent"
+  alt="Quickstart dashboard for the Prometheus agent"
+  src={infrastructurePrometheusIoDashboard}
+/>
+
+<figcaption>
+  Install the quickstart containing the [dashboard for the Prometheus agent](https://newrelic.com/instant-observability/prometheus-agent).
+</figcaption>
+
+## Configure it in Kubernetes [#kubernetes-config]
+
+To configure the agent when using Helm, you should set up your `values.yaml` in one of these two ways:
```

Comment:
I would add 1-2 lines more explaining which are these ways we are referring to

Reply:
Not sure what are you talking about

Reply:
It was moved to the [setup section](https://github.com/newrelic/docs-website/pull/10188/files#diff-bd346b235881db3b874909c7d32eda2fa0fc744994e9ddf697ae875ebb4955a7R111) and it includes more details.